libraries/teams-scenarios/searchBasedMessagingExtension/src/teamsSearchExtensionBot.ts

```diff
@@ -13,7 +13,7 @@ import {
     MessagingExtensionResult,
     TeamsActivityHandler,
     TurnContext
-} from 'botbuilder';
+} from '../../../botbuilder';
```

Comment:
Do we need this change? from importing `botbuilder` to `../../../botbuilder`

Reply:
I reverted it. Thanks!